### ISSUE TITLE
neuprint_common_connectivity broken

### ISSUE THREAD

**Issue by romainFr** [natverse/neuprintr#67]
Fails with 
```
Error in names(x) <- paste("V", seq_along(x), sep = "") : 
  'names' attribute [1] must be the same length as the vector [0]
```

**Comment by jefferis** [comment]
Can you give an example? Is it for any input?

**Comment by jefferis** [comment]
@romainFr is this related to change of dataset to `hemibrain:v1.0.1`?

**Comment by romainFr** [comment]
My mistake, it is actually working. I got confused because it only returns partners common to all inputed neurons and I was looking for partners common to some (at least two) of the inputed neurons.

**Comment by mmc46** [comment]
Sorry to reopen, but then this 

> it only returns partners common to all inputed neurons and I was looking for partners common to some (at least two) of the inputed neurons.

is different behaviour to neuPrint Explorer, where common partners to a subset of bodyids are returned.

Just wanted to double check it is correct.

**Comment by romainFr** [comment]
Actually for me now neuprint explorer returns all the partners of all the neurons (whether any of those partners are in common or not), which is not super useful. Maybe we could have an ALL/ANY option in neuprintr_common_connectivity to switch between different behaviors?

**Comment by alexanderbates** [comment]
I don't really use this function very much, nor am I sure that others in our group do. So feel free to update it to work sensibly. The behaviour does not need to be the same a neuPrint explorer, if whatever we go with is more useful and well documented.

**Comment by jefferis** [comment]
I also agree that requiring partners to have connections to all the query neurons is not the behaviour I would expect.  However more importantly, the `prepost` argument again seems to be backwards. i.e. you specify pre and you get the outputs.
